test/test_utils.py

```diff
@@ -960,11 +783,166 @@ def test_build_container_default_gcc(tmpdir):
         use_host_conda_bld=True,
         image_build_dir=image_build_dir,
     )
+
+    pkg_paths = utils.built_package_paths(r.recipe_dirs['one'])
     build_result = build.build(
         recipe=r.recipe_dirs['one'],
         recipe_folder='.',
-        env={},
+        pkg_paths=pkg_paths,
         docker_builder=docker_builder,
         mulled_test=False,
     )
     assert build_result.success
+
+
+def test_conda_forge_pins(caplog):
+    caplog.set_level(logging.DEBUG)
+    r = Recipes(
+        """
+        one:
+          meta.yaml: |
+            package:
+              name: one
+              version: 0.1
+            requirements:
+              run:
+                - zlib {{ zlib }}
+        """, from_string=True)
+    r.write_recipes()
+    build_result = build.build_recipes(
+        r.basedir,
+        config={},
+        packages="*",
+        testonly=False,
+        force=False,
+        mulled_test=False,
+    )
+    assert build_result
+
+    for k, v in r.recipe_dirs.items():
+        for i in utils.built_package_paths(v):
+            assert os.path.exists(i)
+            ensure_missing(i)
+
+
+def test_bioconda_pins(caplog):
+    """
+    htslib currently only provided by bioconda pinnings
+    """
+    caplog.set_level(logging.DEBUG)
+    r = Recipes(
+        """
+        one:
+          meta.yaml: |
+            package:
+              name: one
+              version: 0.1
+            requirements:
+              run:
+                - htslib {{ htslib }}
```

Comment:
@daler from where do you have this `{{ htslib }}` syntax? It should work without it, does it not?

Reply:
Thanks, yes it seems to work. Was not clear from the docs that this would be true. I was following the lead of [this example](https://conda.io/docs/user-guide/tasks/build-packages/variants.html#general-pinning-examples) where they specify `boost {{ boost }}`. Is that syntax just for when there are multiple variants specified?